invenio_app_ils/holidays/api.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""ILS holidays API."""
+
+from datetime import timedelta
+
+import arrow
+
+from invenio_app_ils.proxies import current_app_ils
+
+_day_increment = timedelta(days=1)  # Atomic increment
```

Comment:
Very minor: I personally tend to use uppercase for constats:
```suggestion
_ONE_DAY_INCREMENT = timedelta(days=1)  # Atomic increment
```